examples/cms/src/FieldMapping.tsx

```diff
@@ -0,0 +1,225 @@
+import { ManagedCollectionField, framer } from "framer-plugin"
+import { useState, useMemo } from "react"
+import { Fragment } from "react/jsx-runtime"
+import { DataSource, FieldConfig, computeFieldConfigs } from "./data"
+import { isNotNull } from "./utils"
+
+interface FieldMappingRowProps {
+    fieldConfig: FieldConfig
+    isIgnored: boolean
+    onFieldToggle: (fieldId: string) => void
+    onFieldNameChange: (fieldId: string, name: string) => void
+}
+
+function FieldMappingRow({ fieldConfig, isIgnored, onFieldToggle, onFieldNameChange }: FieldMappingRowProps) {
+    const field = fieldConfig.field
+
+    const isUnsupported = !field
+    const hasFieldNameChanged = field!.name !== fieldConfig.name
+    const fieldName = hasFieldNameChanged ? field!.name : ""
+    const placeholder = isUnsupported ? "Unsupported Field" : fieldConfig.name
+
+    const isDisabled = useMemo(() => isUnsupported || isIgnored, [isUnsupported, isIgnored])
+
+    return (
+        <Fragment>
```

Comment:
Are we using `<Fragment>` instead of `<>` on purpose?

Reply:
No, this is a remnant from when we split the component. We can update it to use the shorthand <> syntax